tests/test_objective_funs.py

```diff
@@ -72,6 +72,7 @@
     QuasisymmetryTwoTerm,
     RotationalTransform,
     Shear,
+    SurfaceCurrentRegularization,
```

Comment:
can you also add a basic unit test to `test_objective_funs.py::TestObjectiveFunction` that just builds/computes with the new objective for the different field types?